reporting-app/app/policies/task_policy.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+class TaskPolicy < ApplicationPolicy
```

Comment:
There's a `Strata::Task` policy that was created last week as part of the Staff policy work. 
https://github.com/navapbc/oscer/blob/main/reporting-app/app/policies/strata/task_policy.rb

Would using that simplify the scope query? Either way, this should be consolidated